php/WP_CLI/CommandWithTranslation.php

```diff
@@ -125,6 +125,95 @@ public function install( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Updates the active translation of core, plugins, and themes.
+	 *
+	 * [--dry-run]
+	 * : Preview which translations would be updated.
+	 *
+	 * @subcommand update
+	 */
+	public function update( $args, $assoc_args ) {
+
+		// Ignore updates for the default locale.
+		if ( 'en_US' == get_locale() ) {
+			\WP_CLI::line( "Translations updates are not needed for the default locale." );
+
+			return;
+		}
+
+		wp_version_check();  // Check for Core updates.
+		wp_update_themes();  // Check for Theme updates.
+		wp_update_plugins(); // Check for Plugin updates.
+
+		$updates = wp_get_translation_updates(); // Retrieves a list of all translations updates available.
+
+		if ( empty( $updates ) ) {
+			\WP_CLI::line( "No translations updates available." );
+
+			return;
+		}
+
+		// Gets a list of all languages.
+		$all_languages = $this->get_all_languages();
+
+		// Formats the updates list.
+		foreach ( $updates as $update ) {
+			if ( 'plugin' == $update->type ) {
```

Comment:
Although not documented, it seems to be the default behavior of the WordPress API.